.ts_pre_commit_config.yaml

```diff
@@ -3,4 +3,6 @@ check-xml: true
 black: true
 flake8: true
 isort: true
-mypy: false
+mypy: true
+ruff: true
```

Comment:
`ruff: true` means you can set `black`, `flake8` and `isort` to false so please do so.